test/start-kubernetes.sh

```diff
@@ -17,15 +17,14 @@ LOCKFILE="${LOCKFILE:-${REPO_DIRECTORY}/_work/start-kubernetes.exclusivelock}"
 LOCKDELAY="${LOCKDELAY:-300}" # seconds
 NODES=( $DEPLOYMENT_ID-master
         $DEPLOYMENT_ID-worker1
-        $DEPLOYMENT_ID-worker2
-        $DEPLOYMENT_ID-worker3)
+        $DEPLOYMENT_ID-worker2)
```

Comment:
Passing this Via CI environment is more appropriate in my opinion, than hardcoding in script.

Reply:
Same as for the number of CPUs: why would we want a larger configuration on a desktop than in the CI?